### ISSUE TITLE
[Feature request ] Midi output in parallel of CV output

### ISSUE THREAD

**Issue by labodezao** [BelaPlatform/gliss-firmware#22]
Hi, I am developing hybrid acoustic and midi accordion  for handicapted people and i would like that gliss to be a part to control midi CC. 
Actuaklly the outs are MidiCV and I would like to have midi out messages in parallel.

The aim is to control internal effets I developped with some custom made boards and to use gliss as a mod wheel /  bend wheel and also control some acuators in the accordion like sourninas, etc.
Do you think it could be possible to add this function to the firmware / hardware project ?
I would help it i could !

**Comment by mxa** [comment]
Have you considered using a Trill BAR or Trill FLEX Slider for that purpose?

**Comment by labodezao** [comment]
Yes, but I really need Cv output + all the features implemented in Gliss, but I could also use a CAN to convert CV to midi signal in my midi board system if it's too difficult to implement, then I could have both.

**Comment by giuliomoro** [comment]
@labodezao Do you need the output to send the current touch information or the same value as the CV out ?

**Comment by labodezao** [comment]
I need that the Cv output same as it is implemented now and in parallel that the CV 1 and Cv 2 output to converted and send via midi over CC events over midi usb or best midi din 5-socket (or 2 pin head that I could root manually wire inside my system to midi In of my boards.
So the midi CC sould also follow all the function like buffer, on fly record and so on :)

**Comment by giuliomoro** [comment]
(late reply) : @labodezao if you plug gliss via USB to a host device, you can get MIDI through it. What's implemented so far is mostly SYSEX for loading/storing settings and updating the firmware. It wouldn't be hard to add a, say, CCs sent at 100Hz that would contain a sampled copy of the CV outs.
